pingpong-go/src/pingpong/pong/Pong.go

```diff
@@ -1,29 +1,35 @@
-package pong 
+package pong
 
 import (
+	"bytes"
 	"fmt"
 	"net/http"
-	"github.com/gorilla/mux"
 	"strconv"
-	"bytes"
+
+	"github.com/gorilla/mux"
 )
 
 func pongHandler(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
-	i, _ := strconv.Atoi(params["length"])
+	i, err := strconv.Atoi(params["length"])
+	if err != nil {
+		panic(err)
+	}
 	i -= 4
-	var result bytes.Buffer
-	result.WriteString("po")
-	for ; i > 0; i-- { result.WriteString("o") }
+	result := bytes.NewBufferString("po")
```

Comment:
Sorry, I did not your question before. 
I have to correct myself: 
The type varies slightly: In your implementation it is bytes.Buffer in mine it is *bytes.Buffer. But this does not affect the rest of the code, since in Go Methods can be called from the types directly or pointers pointing on types.
Building the code worked fine for me.